server/container_create.go

```diff
@@ -634,12 +634,12 @@ func isInCRIMounts(dst string, mounts []*pb.Mount) bool {
 }
 
 func (s *Server) setupSeccomp(ctx context.Context, specgen *generate.Generator, profile string) error {
-	if profile == "" {
-		// running w/o seccomp, aka unconfined
-		specgen.Config.Linux.Seccomp = nil
-		return nil
-	}
 	if s.Config().Seccomp().IsDisabled() {
+		if profile == "" {
```

Comment:
The test in `ctr_seccomp.bats` is now failing as well, I think we have to change it too.

Reply:
http://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtimemptye/v1alpha2/api.proto#L282